### PR TITLE
NR-34617: Added new quickstart- ADO.NET

### PR DESCRIPTION
**Pull request by dikshab0202**
Ticket: https://issues.newrelic.com/browse/NR-34617 
Description: This PR has the new quickstart integration named "Ado.net"

image

# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: dashboards/ado-dotnet/ado-dotnet.json [hunk 1]
```diff
@@ -0,0 +1,1193 @@
+{
+  "name": "Dotnet - with c# & ado.net",
```
Comment (pkosanam) [inline dashboards/ado-dotnet/ado-dotnet.json:2]: Please change this to "ADO.NET"

**Review by pkosanam — CHANGES_REQUESTED** [review]

**Comment by pkosanam** [comment]
Hi @akristen , Here is the new quickstart integration of ADO.NET
Requesting your review on this IO page content
Thank you
cc: @mdumpati

**Comment by aswanson-nr** [comment]
Hey all, just going to mark this as a draft since it is still waiting on content feedback. Just don't want it to get merged accidentally.

**Comment by raahulbasu** [comment]
1. Add a full stop symbol for each bullet points mentioned under 'why monitor ADO.NET' and 'what's included in this quickstart'
2. Under what's included in this quickstart, bullet point #2 add appropriate spaces, uppercase/lowercase for metric names i.e., HardConnectsPerSecond, HardDisconnectsPerSecond etc.

File: quickstarts/dotnet/ado-dotnet/config.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - Inaccessible databases or issues with the network library.
+    - Database and client software schema incompatibility.
+    - Problems occur when different ADO.NET components interact with one another or your own components.
+    - Whether hard-coded or created by an application, incorrect SQL.
+    - Flawed programming logic.
+
+  ## What’s included in this quickstart?
+
+  The ADO.NET monitoring quickstart from New Relic provides immediate full-stack observability:
+    - Alerts (High load connection, Hard connects exceeding max limit, Pool size exceeding optimum limit).
```
Comment (akristen) [inline quickstarts/dotnet/ado-dotnet/config.yml:21]:
```suggestion
    - Alerts (high load connection, hard connects exceeding max limit, pool size exceeding optimum limit).
```

File: quickstarts/dotnet/ado-dotnet/config.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    - Dashboards (Hard connects per second, Hard disconnects per second, Number of active connections, Soft connects per second,  and more).
```
Comment (akristen) [inline quickstarts/dotnet/ado-dotnet/config.yml:22]:
```suggestion
    - Dashboards (hard connects per second, hard disconnects per second, number of active connections, soft connects per second,  and more).
```

File: quickstarts/dotnet/ado-dotnet/config.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    - Alerts inform end users about the quality of their applications on a proactive basis.
+    - Monitor web transactions.
+
+
+summary: |
+  Discover more about ADO.NET, the importance of New Relic's ADO.NET quickstart, the advantages of ADO.NET monitoring, and the best features to look for in this quickstart.
```
Comment (akristen) [inline quickstarts/dotnet/ado-dotnet/config.yml:28]:
```suggestion
  Discover how the ADO.net quickstart gives you an advantage with comprehensive monitoring capabilities.
```

File: quickstarts/dotnet/ado-dotnet/config.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    - Database and client software schema incompatibility.
+    - Problems occur when different ADO.NET components interact with one another or your own components.
+    - Whether hard-coded or created by an application, incorrect SQL.
+    - Flawed programming logic.
+
+  ## What’s included in this quickstart?
+
+  The ADO.NET monitoring quickstart from New Relic provides immediate full-stack observability:
+    - Alerts (high load connection, hard connects exceeding max limit, pool size exceeding optimum limit).
+    - Dashboards (hard connects per second, hard disconnects per second, number of active connections, soft connects per second,  and more).
```
Comment (pkosanam) [inline quickstarts/dotnet/ado-dotnet/config.yml:22]: Observed extra space after "second,"

I feel, text need to be like below:
Dashboards (hard connects per second, hard disconnects per second, number of active connections, soft connects per second and more).

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: quickstarts/dotnet/ado-dotnet/config.yml [hunk 6]
```diff
@@ -0,0 +1,53 @@
+
+slug: ado-dotnet
+description: |
+  ## Comprehensive monitoring quickstart for ADO.net
```
Comment (pkosanam) [inline quickstarts/dotnet/ado-dotnet/config.yml:4]: Should this be ADO.net or ADO.NET ?

File: quickstarts/dotnet/ado-dotnet/config.yml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+  When you monitor ADO.net with New Relic, you can observe your ADO.NET data set in real time. Our .NET agent will also let you correlate transactions moving across your application environment and troubleshoot while working in a live development environment.
```
Comment (pkosanam) [inline quickstarts/dotnet/ado-dotnet/config.yml:6]: Should this be ADO.net or ADO.NET ?

File: quickstarts/dotnet/ado-dotnet/config.yml [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+  The ADO.NET monitoring quickstart from New Relic provides immediate full-stack observability:
+    - Alerts (high load connection, hard connects exceeding max limit, pool size exceeding optimum limit).
+    - Dashboards (hard connects per second, hard disconnects per second, number of active connections, soft connects per second and more).
+    - Alerts inform end users about the quality of their applications on a proactive basis.
+    - Monitor web transactions.
+
+
+summary: |
+  Discover how the ADO.net quickstart gives you an advantage with comprehensive monitoring capabilities.
```
Comment (pkosanam) [inline quickstarts/dotnet/ado-dotnet/config.yml:28]: Should this be ADO.net or ADO.NET ?

**Review by pkosanam — CHANGES_REQUESTED** [review]

**Review by pkosanam — APPROVED** [review]

**Review by pkosanam — APPROVED** [review]
looks good

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**